dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/CommandMapper.java

```diff
@@ -50,12 +52,15 @@ List<CommandCount> countCommandState(
      *
      * @return
      */
-    List<Command> queryCommandPage(@Param("limit") int limit, @Param("offset") int offset);
+    IPage<Command> queryCommandPage(Page<Command> page);
 
     List<Command> queryCommandByIdSlot(@Param("currentSlotIndex") int currentSlotIndex,
                                        @Param("totalSlot") int totalSlot,
                                        @Param("idStep") int idStep,
                                        @Param("fetchNumber") int fetchNum);
 
     void deleteByWorkflowInstanceIds(@Param("workflowInstanceIds") List<Integer> workflowInstanceIds);
+
+    IPage<Command> queryCommandPageByIds(Page<Command> page, @Param("definitionCodes") List<Long> definitionCodes,
+                                         @Param("projectCode") Long projectCode);
```

Comment:
```suggestion
    IPage<Command> queryCommandPageByIds(Page<Command> page, @Param("workflowDefinitionCodes") List<Long> workflowDefinitionCodes);
```

Please remove the projectCode at this method, this param is never used?
And use more accurate naming for definitionCodes, since there exist workflowDefinitionCodes and taskDefinitionCodes